acme/acme/jose/jwk_test.py

```diff
@@ -114,6 +115,12 @@ def test_init_auto_comparable(self):
             self.jwk256_not_comparable.key, util.ComparableRSAKey))
         self.assertEqual(self.jwk256, self.jwk256_not_comparable)
 
+    def test_init_default_generate(self):
+        from acme.jose.jwk import JWKRSA
+        jwk = JWKRSA()
+        # pylint: disable=protected-access
+        self.assertTrue(isinstance(jwk.key._wrapped, rsa._RSAPrivateKey))
```

Comment:
You want to check if it's an instance of `cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateKey`, which is the interface; you don't want to check against the private implementation.